api/src/main/java/com/datatorrent/api/StatsListener.java

```diff
@@ -113,6 +113,8 @@
     long getLatencyMA();
 
     List<OperatorResponse> getOperatorResponse();
+
+    String getOperatorName();
```

Comment:
stat listener only have access to the BatchedOperatorStats, hence added it as part of it. I am closing this pull request now. Making changes as per discussion on the dev thread.